pkg/controller/kubefedcluster.go

```diff
@@ -1,15 +1,19 @@
 package controller
 
 import (
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
 	"github.com/codeready-toolchain/toolchain-common/pkg/controller"
+	"github.com/go-logr/logr"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
 	"k8s.io/klog"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/kubefed/pkg/controller/kubefedcluster"
 	"sigs.k8s.io/kubefed/pkg/controller/util"
 )
 
-func StartKubeFedClusterControllers(mgr manager.Manager, stopChan <-chan struct{}) error {
+func StartKubeFedClusterControllers(log logr.Logger, mgr manager.Manager, stopChan <-chan struct{}) error {
+	cluster.EnsureKubeFedClusterCrd(log, mgr, stopChan)
```

Comment:
As I suggested in https://github.com/codeready-toolchain/toolchain-common/pull/44#discussion_r332679480 we don't need to create the CRD in the go routine. So we can return an error in `EnsureKubeFedClusterCrd()` so we can check it here.

Reply:
done in eeec511